### DISCUSSION TITLE
ntfy Push Nachrichten

### DISCUSSION THREAD

**Discussion by modezz** [wiggal/GEN24_Ladesteuerung#53] [Ideas]
Servus Wiggal,

man bekommt j per Push immer die Variable _newpercent_ gesendet, ev sollte man diese anpassen auf _newPercent/100,"%"_ oder gleich in kombination mit _aktuellerLadewert_ ? nur zum besseren Verständnis, ausser nur ich verstehs wieder mal nicht...

Danke & SG

**Comment by wiggal** [comment]
Ok stimmt,
ändere ich bei Gelegenheit
Schönen Gruß

**Comment by wiggal** [comment]
Ich habe es gleich angepasst, dann vergess ich es nicht ;-)

`Am WR geschrieben: 48.82% = 5000W`
